src/java/org/apache/cassandra/schema/MemtableParams.java

```diff
@@ -110,6 +112,22 @@ public static MemtableParams get(String key)
         }
     }
 
+    public static MemtableParams getWithFallback(String key)
+    {
+        try
+        {
+            return get(key);
+        }
+        catch (ConfigurationException e)
+        {
+            LoggerFactory.getLogger(MemtableParams.class).error("Invalid memtable configuration \"" + key + "\" in schema. " +
```

Comment:
nit: Any reason we create the logger inline? (Looks like it would be the only usage, but just curious...)

Reply:
It's never used in other places, and this is a rare event which to me did not warrant always creating the logger object on its own.